wallet/src/wallet/tests.rs

```diff
@@ -1251,6 +1275,112 @@ fn locked_wallet_cant_sign_transaction(#[case] seed: Seed) {
             .unwrap();
     }
 }
+
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn locked_wallet_standalone_keys(#[case] seed: Seed) {
```

Comment:
1) I'd say the emphasis here is on "standalone_keys" and not "locked_wallet".
2) Do we still need `test_add_standalone_private_key`? It looks like it does only a subset of what this test does. (I guess the only thing this test doesn't do is check the result of `wallet.get_transaction`, but we could add the check).
3) If it's still needed, let's put the two tests near each other.